packages/nimble/R/cppDefs_core.R

```diff
@@ -7,13 +7,11 @@
 ## Base class for C++ file and compilation information
 cppDefinition <- setRefClass('cppDefinition', 
                              fields = list(
-                                 filename = 'ANY',	#  character, what filename (to which .h and .cpp will be appended) is this definition in
-                                 CPPusings = 'ANY',	#  character
```

Comment:
Here's a random idea that's outside the scope of this PR: what if we set these types to something like
```
CPPusings = if(checkNimbleTypes) 'character' else 'ANY'
```
and then set a global variable `checkTypes = FALSE` or ` = (length(Sys.getenv('NIMBLE_CHECK_TYPES')) > 0)`, so that we can check types while developing code that might break type checking, but avoid the overhead in general use. @perrydv What do you think?